docs/answerfile.md

```diff
@@ -328,4 +334,4 @@ Your answer file can also be used to upgrade your machines. Here is an example:
 </installation>
 ```
 
-As you can see, `mode` is set on `upgrade`. Be sure to target the right disk to search for previous existing installations (here `sda`). Do NOT specify `primary-disk` and `guest-disk`!
\ No newline at end of file
+As you can see, `mode` is set on `upgrade`. Be sure to target the right disk to search for previous existing installations (here `sda`). Do NOT specify `primary-disk` and `guest-disk`!
```

Comment:
This line should not have been changed. (I suppose your text editor found there was no newline at end of file and added one)